pkgs/os-specific/darwin/CoreSymbolication/default.nix

```diff
@@ -1,8 +1,33 @@
-{ lib, fetchFromGitHub, fetchpatch, stdenv }:
+{
+  lib,
+  fetchFromGitHub,
+  fetchpatch,
+  stdenvNoCC,
+  darwin-stubs,
+  darwin,
+}:
 
-stdenv.mkDerivation {
-  pname = "core-symbolication";
-  version = "unstable-2018-06-17";
+let
+  inherit
+    (
+      if stdenvNoCC.isx86_64 then
+        {
+          version = "10.12";
+          stubs = darwin-stubs;
+        }
```

Comment:
can this be defined another way then using the architecture? seems like it will be problematic if we try to get the x64 and aarch64 to be unified.

Reply:
Sure. I can push the stubs to the SDK, so they can be different depending on the SDK. Currently, darwin.darwin-stubs is hardcoded to the 10.12 SDK. For the 11.0 SDK, I just link the ones from the SDK itself, but they can eventually be replaced by generated stubs. That would let me use the stubs unconditionally in CoreSymbolication.

Reply:
I force-pushed an update to add the stubs to the SDK and updated CoreSymbolication to use them.